docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/linux-packages.rst

```diff
@@ -76,19 +76,28 @@ Install PostgreSQL using ``apt`` or ``yum``
       sudo systemctl start postgresql.service
       sudo systemctl enable postgresql.service
 
-#. Configure a system account that Determined will use to connect to PostgreSQL. For example, to use
-   the default ``postgres`` user but update its password:
+#. The authentication methods enabled by default may vary depending on the provider of your
+   PostgreSQL distribution. Ensure that an appropriate authentication method is configured in
+   ``pg_hba.conf`` to enable the ``determined-master`` to connect to the database.
 
-   .. code::
+   When configuring the database connection (:ref:`configure_the_cluster`):
 
-      sudo -u postgres psql postgres
-      postgres=# \password postgres
+   -  If you specify the ``db.hostname`` property, a PostgreSQL ``host`` (TCP/IP) connection will be
+      required.
+   -  If you omit the ``db.hostname`` property, a PostgreSQL ``local`` (Unix-domain socket)
+      connection will be required.
 
-#. Finally, create a database for Determined's use.
+#. Finally, create a database for Determined's use and configure a system account that Determined
+   will use to connect to the database. For example, the following commands will create a database
+   named ``determined``, a user named ``determined`` with the password ``determined-password``, and
+   then grants the user access to the database:
```

Comment:
Sorry, I hate to be That Grammar Guy, but this has inconsistent verb tense in the list. :)

Probably the easiest fix is "will create, will create, will then grant"
```suggestion
   will use to connect to the database. For example, the following commands will create a database
   named ``determined``, create a user named ``determined`` with the password ``determined-password``, and
   then grant the user access to the database:
```

Reply:
Grammerly.com was happy, but yeah grants -> will grant is more consistent.